tempodb/encoding/v2/appender.go

```diff
@@ -4,7 +4,7 @@ import (
 	"hash"
 	"sync"
 
-	"github.com/cespare/xxhash"
+	"github.com/cespare/xxhash/v2"
```

Comment:
we are using New from this package here and looks like it changed in v2, can you verify this is same, maybe a go playground link would be helpful to see if the behaviour is same:

[cespare/xxhash@`v1.1.0...v2.3.0`#diff-2e98786ac7](https://github.com/cespare/xxhash/compare/v1.1.0...v2.3.0#diff-2e98786ac7a2943113c4902a927fe34aa0f301737dd8e29893afe6991378e8e8R39)

Reply:
Yes, they are the same. See https://go.dev/play/p/ZXuwERoBlEi  

The differences are:  

1. `New()` in v1 returns a `hash.Hash64` interface. In v1, it is not possible to set a seed.  
2. `New()` in v2 returns a `Digest` type that implements the `hash.Hash64` interface. As shown in the GoDoc(https://pkg.go.dev/github.com/cespare/xxhash/v2#Digest), the `Digest` type allows setting a seed using `NewWithSeed()` if needed. Otherwise, you can use `New()`, which behaves the same as in v1.